__tests__/app.test.js

```diff
@@ -133,6 +133,52 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST /api/articles/:article_id/comments", () => {
+  const newComment = { username: "butter_bridge", body: "dolor. Consequatur quasi itaque culpa. Tempora ut autem est ad est" };
+  test("status 201, should insert one comment according to article_id", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+        expect(comment).toHaveProperty("author", expect.any(String));
+        expect(comment).toHaveProperty("body", expect.any(String));
+        expect(comment).toHaveProperty("article_id", expect.any(Number));
+      });
+  });
+  test("status 404, should return obj with message of id is not found", () => {
+    return request(app)
+      .post("/api/articles/9999/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+  test("status 400: invalid id ", () => {
+    return request(app)
+      .post("/api/articles/banana/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+
+  test("400: returns if any of the required properties for posting is missing", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({ body: "dolor. Consequatur quasi itaque culpa. Tempora ut autem est ad est" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
```

Comment:
Nice work - could we also include a test for username not found (404) so if the username is something random :)